.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,10 @@
-<!--
+## Description
 
-Please add a reference to the issue that this PR addresses and indicate which
-files are most critical to review. If it fully addresses a particular issue,
-please include "Closes #XXX" (where "XXX" is the issue number).
+Please provide a brief description of your changes and reference any related issues.
 
-If this PR is non-trivial/large/complex, please ensure that you have either
-created an issue that the team's had a chance to respond to, or had some
-discussion with the team prior to submitting substantial pull requests. The team
-can be reached via GitHub Discussions or the Cosmos Network Discord server in
-the #cometbft channel. GitHub Discussions is preferred over Discord as it
-allows us to keep track of conversations topically.
-https://github.com/cometbft/cometbft/discussions
+## Checklist
```

Comment:
let's delete this and the checklist as well

the only thing to keep in the contributing guidlines